README.adoc

```diff
@@ -331,11 +331,11 @@ Arch Linux, among other Linux distributions may have the user namespace in the k
 * *_Failed to read the 'rules' property from 'CSSStyleSheet': Cannot access rules_*
 +
 Starting Chromium 64, accessing CSS rules in a stylesheet loaded from the local filesystem violates a CORS policy.
```

Comment:
I'd suggest to change:

"_loaded from the local filesystem violates a CORS policy_"

With:

"_loaded from the local filesystem or an external location violates CORS policies_"